Exercises/chapter-10.txt

```diff
@@ -0,0 +1,7 @@
+1. Why is it important to protect both the edit and update actions?
+Because these are actions that can change data in your database.
```

Comment:
the `edit` route doesn't actually change data.  but it does *show* data, and you don't want just anybody looking up your skirt!